data/top_model/abp_output_transform.txt

```diff
@@ -1,125 +1,55 @@
+00:00:10:000              5                   5                  NA                 out      generator_con
+00:00:20:000              1                   1                  NA       packetSentOut            sender1
+00:00:20:000             11                   1                   1             dataOut            sender1
+00:00:23:000             11                   1                   1                 out            subnet1
+00:00:33:000              1                   1                   1                 out          receiver1
+00:00:36:000              1                   1                   1                 out            subnet2
+00:00:36:000              1                   1                   1      ackReceivedOut            sender1
+00:00:46:000              2                   2                   1       packetSentOut            sender1
+00:00:46:000             20                   2                   0             dataOut            sender1
+00:00:49:000             20                   2                   0                 out            subnet1
+00:00:59:000              0                   0                   0                 out          receiver1
+00:01:02:000              0                   0                   0                 out            subnet2
+00:01:02:000              0                   0                   0      ackReceivedOut            sender1
+00:01:12:000              3                   3                   0       packetSentOut            sender1
+00:01:12:000             31                   3                   1             dataOut            sender1
+00:01:15:000             31                   3                   1                 out            subnet1
+00:01:25:000              1                   1                   1                 out          receiver1
+00:01:28:000              1                   1                   1                 out            subnet2
+00:01:28:000              1                   1                   1      ackReceivedOut            sender1
+00:01:30:000              4                   4                   1                 out      generator_con
+00:01:40:000              4                   4                   1       packetSentOut            sender1
+00:01:40:000             40                   4                   0             dataOut            sender1
+00:01:43:000             40                   4                   0                 out            subnet1
+00:01:53:000              0                   0                   0                 out          receiver1
+00:01:56:000              0                   0                   0                 out            subnet2
+00:01:56:000              0                   0                   0      ackReceivedOut            sender1
+00:02:06:000              5                   5                  NA       packetSentOut            sender1
```

Comment:
If you compare with other packet sentout it is different

Reply:
bug has be fixed.